src/NaturalSort/NaturalSort.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Buildium.NaturalSort</id>
+    <version>1.0.0</version>

```

Comment:
This project seemed to make sense to have a version 1.0.0, since its functionality was already tested with Buildium.